app/controllers/admin/dashboards_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Admin::DashboardsController < Admin::ApplicationController
+  def index
+    @slide_count = Slide.count
+    @user_count = User.count
+    @conversion_failed_count = Slide.failed.count
+    @comment_count = Comment.count
+
+    s = Slide.arel_table
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 's' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->